css/keyui.sass

```diff
@@ -140,6 +140,14 @@
         border: 1px solid darken($n, 15%)
         background-color: lighten($n, 15%)
         box-shadow: 0 0px 3px darken($n, 10%) inset
+        &:valid
+            padding-right: 5px
+            padding-left: 25px
+            background: url('../img/x.png'),url('../img/check.png')
```

Comment:
Bitmap images aren't scalable and their size is relativly big comparing to code only approach. Both these images have 50KB of size. You could code this check and X mark using content css property on ```:after``` or ```:before``` pseudoclass, or border transforms approach like I did with check mark. 

Examples:
https://codepen.io/fmobiledesigns/pen/wMjYOm (use dev tools to see css and html structure).
- checkmark used in test (run by using gulp watch command)

Reply:
Ohhhhh. All right. I didnt know you could do an after and before inside an input.